WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.java

```diff
@@ -59,8 +63,9 @@ public void onClick(View v) {
                             SignInFragment.ENTERED_URL_KEY));
                     HelpshiftHelper.getInstance().addMetaData(MetadataKey.USER_ENTERED_USERNAME, extras.getString(
                             SignInFragment.ENTERED_USERNAME_KEY));
+                    origin = (Tag) extras.get(HelpshiftHelper.ORIGIN_KEY);
```

Comment:
Should this (and line 78 below) check `extras.containsKey(HelpshiftHelper.ORIGIN_KEY)` before reading `origin` to ensure `origin` is never set to null?